src/Domain/CoordinateMaps/CoordinateMap.tpp

```diff
@@ -557,8 +541,9 @@ auto CoordinateMap<SourceFrame, TargetFrame, Maps...>::
                     InverseJacobian<T, dim, SourceFrame, TargetFrame>,
                     Jacobian<T, dim, SourceFrame, TargetFrame>,
                     tnsr::I<T, dim, TargetFrame>>{
-      tnsr::I<T, dim, TargetFrame>(std::move(mapped_point)), std::move(inv_jac),
-      std::move(jac), std::move(frame_velocity)};
+      tnsr::I<T, dim, TargetFrame>(std::move(mapped_point)),
+      determinant_and_inverse(jac).second, std::move(jac),
+      std::move(frame_velocity)};
```

Comment:
Add in the `Coordinatemap.hpp` docs for this function that we don't call the individual `inverse_jacobian` functions and multiply them together, but rather just compute the jacobian and numerically invert it because most of our production time-dependent maps do this already.

Reply:
Is there a reason to keep the `inverse_jacobian` implementations on the individual maps at all?  Doesn't need to be done in this PR, but removing them would be nicer than documenting that they are unused.

Reply:
Well they are still used in the individual `inverse_jacobian` function of `CoordinateMap{Base}`, correct? Whether or not that function is actually used in practice, I don't know. If not, then yes we can probably get rid of it

Reply:
Or we could make it do the numerical inversion like this function now does.

Reply:
Yeah we could! Let's discuss on the Monday call

Reply:
Another thing for a separate PR: this function could also return the Jacobian determinant. We need this a bit in the hydro cases.

Reply:
From the Monday call: We can just numerically invert the jacobian in the individual function as well which means we don't need the individual `inverse_jacobian` functions. See #6233